.github/docker/testing/Dockerfile.testing-plugins-bookworm

```diff
@@ -0,0 +1,47 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/debian:bookworm
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# fix locale
+RUN bash -e <<EOF
+
+apt-get update
+apt-get install -y locales libcurl4-openssl-dev curl wget zstd jq
+rm -rf /var/lib/apt/lists/*
+localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+apt-get clean
+
+EOF
+
+ENV LANG en_US.utf8
+
+RUN bash -e <<EOF
+apt-get update
+# Install Robotframework
+apt-get install -y python3-dev python3-pip
+rm -rf /usr/lib/python3.11/EXTERNALLY-MANAGED
```

Comment:
@Evan-Adam what is this

Reply:
needed to install library with pip, without it we have this error 
`
63.32 error: externally-managed-environment
63.32
63.32 × This environment is externally managed
63.32 ╰─> To install Python packages system-wide, try apt install
63.32     python3-xyz, where xyz is the package you are trying to
63.32     install.
63.32
63.32     If you wish to install a non-Debian-packaged Python package,
63.32     create a virtual environment using python3 -m venv path/to/venv.
63.32     Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
63.32     sure you have python3-full installed.
63.32
63.32     If you wish to install a non-Debian packaged Python application,
63.32     it may be easiest to use pipx install xyz, which will manage a
63.32     virtual environment for you. Make sure you have pipx installed.
63.32
63.32     See /usr/share/doc/python3.11/README.venv for more information.
63.32
63.32 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
63.32 hint: See PEP 668 for the detailed specification.
`